content/vendor/guides/configuring-application-snapshots.md

```diff
@@ -171,7 +171,7 @@ The first step is to add a backup resource file.
 Like the Postgres StatefulSet Definition file, the file contents are available in the [Appendix A]
 Create a new release and add the [backup.yaml](#backupyaml) file to the application manifests.
 This contains the minimal configuration and will suffice.
-For further details on all of the available specs, please check the Velero documentation [here](https://velero.io/docs/v1.3.2/api-types/backup/)
+For further details on all of the available specs, please check the Velero documentation [here](https://velero.netlify.app/docs/v1.3.2/api-types/backup/)
```

Comment:
Is this where their official documentation lives? Pointing to a netlify app is all sorts of weird.

Reply:
It's in the thread here. Their site has been down for quite some time, and they're recommending you use this site for now. No indication of when it will get fixed, so we're moving over the links for now. https://replicated.slack.com/archives/C02BPJQQP28/p1643749416745029

Reply:
Gotcha. My only suggestion then would be to make that a separate PR from the 1.61 since they're unrelated changes. But sure, whatever.